doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -189,6 +207,36 @@ Reading data from a Neo4j Database can be done in 3 ways:
  * with a set of node Labels 
  * by specifying a relationship
 
+=== Considerations about partitioning
+
+While we're trying to pull off the data we offer the possibility to partition the extraction in order
+parallelizing it.
+
+==== How we parallelize the query execution
+
+Considering that we have three options
+
+1. Node extraction
+2. Relationship extraction
+3. Query extraction
+
+We adopt generally provide a general count on what you're trying to pull of and add build
```

Comment:
provide an example here, this isn't very clear.  For example, I'm loading just Person nodes.  I set partitions=5.  This turns into:

```
MATCH (p:Person) RETURN p LIMIT 20 SKIP 0
MATCH (p:Person) RETURN p LIMIT 20 SKIP 20
...
```

Reply:
In theory we could also leverage cursors:

- given there is a unqiue constraint
- we can return the data ordered (using index backed order by) with a limit
- then we can take the last value returned and add an `WHERE n.id > $cursor` which then would only query from that value
- pure skip limit queries need to re-execute and throw away, so they get slower over time